hooks/templates/service_config_ceph.yaml.j2

```diff
@@ -0,0 +1,34 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Seems to be a change not picked as a move, but this reminds me that maybe we should run the HCI job over this change before merging... @cjeanner @viroel @rebtoor

Reply:
`podified-multinode-hci-deployment-crc` is already running in this PR, is this the job you mean?

Reply:
okok, I didn't spotted it when checking jobs. Yes, it's that one.